cogdl/oag/oagbert_metainfo.py

```diff
@@ -68,14 +70,14 @@ def _convert_token_ids_to_text(self, token_ids):
             return self.tokenizer.convert_tokens_to_string(self.tokenizer.convert_ids_to_tokens(token_ids))
 
     def print_oag_instance(
-        self,
-        input_ids,
-        token_type_ids,
-        input_masks,
-        masked_lm_labels,
-        position_ids,
-        position_ids_second,
-        predictions=None,
+            self,
```

Comment:
Maybe we use different code formatting settings. I suggest not changing indents here.